apps/api/openapi/openapi.json

```diff
@@ -17,10 +17,6 @@
     {
       "url": "http://localhost:3000",
       "description": "Local development server"
-    },
-    {
-      "url": "https://api.example.com",
-      "description": "Production server"
     }
   ],
   "components": {
```

Comment:
See below in `securitySchemas.BearerAuth`.  This entry acts as the specification for the header that contains the bearer token.  When we also add header in the `parameters` of some routes, it is treated as a duplicate during Fern's SDK generation, which results in an error building the typescript SDK.

Reply:
these docs are auto-generated by swagger - is this something we instead need to change in ./src/config/swagger?